go.mod

```diff
@@ -10,10 +10,9 @@ require (
 
 require (
 	github.com/agext/levenshtein v1.2.3 // indirect
-	github.com/apparentlymart/go-textseg/v13 v13.0.0 // indirect
 	github.com/apparentlymart/go-textseg/v15 v15.0.0 // indirect
 	github.com/fatih/color v1.15.0 // indirect
-	github.com/go-resty/resty/v2 v2.9.1 // indirect
+	github.com/go-resty/resty/v2 v2.10.0 // indirect
```

Comment:
Not really an issue, but imo dependency upgrades should either be directly related to the code changes or be done in a different commit. This is to avoid potential merge conflicts.

No need to change, but something too consider for the next PR (I assume you ran `go mod tidy`).